src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesLauncher.java

```diff
@@ -132,6 +133,7 @@ public synchronized void launch(SlaveComputer computer, TaskListener listener) {
             try {
                 pod = client.pods().inNamespace(namespace).create(pod);
             } catch (KubernetesClientException e) {
+                Metrics.metricRegistry().counter("k8s.cloud.pods.creation.failed").inc();
```

Comment:
Should be abstracting the key creation to a method such as `requestMetricNameFor`. No label here, but needs to be abstracted anyway.

Reply:
Thanks for looking into this PR and your suggestions.
Many metric names are static, so I moved then to a separate class and made most of them constants. There are a few that depend on some params, I put them in methods as you suggested.
Let me know if it looks better now and if you have any concern.